packages/jest-cli/src/cli/index.js

```diff
@@ -33,18 +33,22 @@ import {sync as realpath} from 'realpath-native';
 import init from '../lib/init';
 import logDebugMessages from '../lib/log_debug_messages';
 
-export async function run(maybeArgv?: Argv, project?: Path) {
+export const run = async (
+  maybeArgv?: Argv,
+  project?: Path,
+): Promise<?AggregatedResult> => {
+  let results, globalConfig;
   try {
     const argv: Argv = buildArgv(maybeArgv, project);
 
     if (argv.init) {
       await init();
-      return;
+      return Promise.resolve(results);
```

Comment:
```suggestion
      return results;
```

it's an `async` function, no need to wrap in `Promise.resolve`